README.md

```diff
@@ -566,6 +566,20 @@ Accesses a notice's payload, which can be read or filtered. Payload includes:
 notice[:params][:my_param] = 'foobar'
 ```
 
+#### Notice#objects[] & Notice#objects[]=
+
+Each notice can carry arbitrary objects. The `objects` parameter is yielded
+while executing [`add_filter`](#airbrakeadd_filter) callbacks. You can retrieve
+wanted information from attached objects and modify the notice's payload:
+
+```rb
+notice.objects[:my_object] = Object.new
+
+Airbrake.add_filter do |notice, objects|
```

Comment:
It is rather strange that you write data to `notice.objects`, but access it by accepting `objects` in filters. Why user can't read from `notice.object` information he wrote there?